docs/swagger.yml

```diff
@@ -145,13 +160,15 @@ paths:
           description: "Credential retrieved"
           schema:
             $ref: "#/definitions/CredentialOut"
+        401:
+          description: "Not authorized"
         404:
           description: "Credential not found"
     put:
       tags:
         - "Credential"
-      summary: "Update an existing credential"
-      description: "Update a credential for use when connecting to a target system."
+      summary: "Full update an existing credential"
```

Comment:
Should this say 'Fully update an existing credential'?

Reply:
Yep